youths/schema.py

```diff
@@ -122,6 +126,41 @@ class CreateMyYouthProfileInput(YouthProfileFields):
     )
 
 
+class CreateYouthProfileMutation(relay.ClientIDMutation):
+    class Input:
+        service_type = graphene.Argument(AllowedServiceType, required=True)
+        profile_id = graphene.Argument(graphene.ID, required=True)
+        youth_profile = CreateMyYouthProfileInput(required=True)
+
+    youth_profile = graphene.Field(YouthProfileType)
+
+    @classmethod
+    @staff_required(required_permission="manage")
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **input):
+        input_data = input.get("youth_profile")
+        if input.get("service_type") != ServiceType.YOUTH_MEMBERSHIP.value:
+            raise CannotPerformThisActionWithGivenServiceType("Incorrect service type")
+
+        profile = Profile.objects.get(pk=from_global_id(input.get("profile_id"))[1])
```

Comment:
one trick that I learned is you could do:
```python
profile = graphene.Node.get_node_from_global_id(info, input.get("profile_id"), only_type=ProfileNode)
```
more reliable way to get the object 🙂

Reply:
I'm probably not willing to import `ProfileNode` in youth schema because `YouthProfileType` is already imported in profile schema (too circular). There will be huge changes in this when youth profile is going to be separated from profile backend 😄

Reply:
alright, fair enough 🙂